===================================================================
RCS file: /cvs/gnome/gconf/gconf/gconf.c,v
retrieving revision 1.150
retrieving revision 1.151
diff -u -r1.150 -r1.151
--- gconf/gconf.c 2005/11/11 13:33:01  1.150
+++ gconf/gconf.c 2005/12/08 13:00:58  1.151
@@ -2749,10 +2749,11 @@
           if (*p == '@')
             {
               /* *p is the @ that ends a seq */
+              char *end_seq;
               guchar val;
               
-              val = strtoul (start_seq, &end, 10);
-              if (start_seq != end)
+              val = strtoul (start_seq, &end_seq, 10);
+              if (start_seq != end_seq)
                 g_string_append_c (retval, val);
               
               start_seq = NULL;
